backend/backend/settings/deps/ledger.py

```diff
@@ -17,7 +17,7 @@
 LEDGER_PEER_TLS_CLIENT_KEY = '/var/hyperledger/tls/client/pair/tls.key'
 LEDGER_PEER_TLS_CLIENT_CERT = '/var/hyperledger/tls/client/pair/tls.crt'
 
-LEDGER_CLIENT_STATE_STORE = '/tmp/hfc-cvs'
+LEDGER_CLIENT_STATE_STORE = os.getenv('LEDGER_CLIENT_STATE_STORE', '/tmp/hfc-cvs')
```

Comment:
Is there a particular advantage to being able to specify the location of the state store?

Reply:
In case user launch the backend in a local way without providing the `LEDGER_CLIENT_STATE_STORE` env variable and to be compatible with older helm chart when using a new substra-backend image :)

Reply:
I think you misread my comment - my question is about the necessity of the `LEDGER_CLIENT_STATE_STORE` env variable. What about forcing the path to an arbitrary value (like `/tmp/hfc-cvs, `/var/hyperledger/hfc-cvs`, etc) and _not_ make it customizable? In other words, what's the value of making it customizable through an env variable?

Reply:
Nope, I understand well.

In a k8s setup, default value in helm chart is `/usr/src/app/hfc-cvs`, because we know that `/usr/src/app` exists in the docker container as it uses substra-backend image.
In a non-docker, non-k8s setup, i. e. in local to debug for instance, we don't know what folder can be used as the state store to write files. It depends on permissions, and where the backend is launch, and if the `LEDGER_CLIENT_STATE_STORE` is defined. If not, it uses the previous default value (`/tmp/hfc-cvs` which should be writable). The value of allowing the user to customize it through an env variable depends on the way the user launch it